cmd/monoctl/get/audit_log.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2021 Monoskope Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package get
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/finleap-connect/monoctl/cmd/monoctl/flags"
+	"github.com/finleap-connect/monoctl/internal/config"
+	"github.com/finleap-connect/monoctl/internal/usecases"
+	authutil "github.com/finleap-connect/monoctl/internal/util/auth"
+	"github.com/spf13/cobra"
+)
+
+func NewGetAuditLogCmd() *cobra.Command {
+	var (
+		from string
+		to string
+
+		layout = "02.01.2006" // don't change. This corresponds to DD.MM.YYYY
```

Comment:
I bet there is a constant in package "time". https://pkg.go.dev/time#pkg-constants

Reply:
By the way, is it only possible to get that audit log for full days then? No time part?

Reply:
believe me, i looked. The problem all of them use the English layout MM.DD.YYYY so I had to manually do the german one.

I made sure to make it extendable so theoretically layout can be extended to support whatever we want (or even a list of layouts is possible). At the end it's converted to a timestamp. So If we want we can support time but also support both date only and DateTime

Reply:
Since we're doing some open-source project here I think having an "English" layout is fine. We can not expect every user of Monoskope to use "German" date format.

Reply:
Ok, the time package specifies format constants for DateTime and leaves it up to the user to define "custom" formats for e.g. dates only. 

I did some more digging and according to [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) the international standard is YYYY-MM-DD as all other variants lead to confusion but this doesn't feel natural (at least for me) and all services I used til today used this format but inverted so I would say DD.MM.YYYY is international enough what do you think?

Reply:
Please let us stick to some standard here and not let us decide based on a gut feeling. To me YYYY-MM-DD feels natural and I use it regularly for stuff since you can sort by it where starting with days leaves you not being able to sort by it. But as I said that's a gut feeling and should not make up our decision here. BUT it is in fact YYYY-MM-DD is a standard (ISO 8601) while DD.MM.YYYY is not AFAIK.

Reply:
Yeah, you are right. Agreed 👍